source/guides/maintenance/developer-workstation.rst

```diff
@@ -17,67 +17,204 @@ other Linux\* distributions.
+
+         * - Machine learning Docker container.
+           - `machine-learning <https://clearlinux.org/software/docker/machine-learning/>`_
+
+         * - Pre-built Python libraries for Data Science.
+           - `python-extras <https://clearlinux.org/software/bundle/python-extras>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+   .. tab:: Computer Vision Engineer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - Build computer vision applications.
+           - `computer-vision-basic <https://clearlinux.org/software/bundle/computer-vision-basic/>`_
+
+         * - Work with deep learning and edge-optimized models.
+           - `computer-vision-models <https://clearlinux.org/software/bundle/computer-vision-models/>`_
+
+         * - Basic OpenVINO™ toolkit.
+           - `computer-vision-openvino <https://clearlinux.org/software/bundle/computer-vision-openvino/>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+         * - All content for pkgconfig file opencv.pc.
+           - `devpkg-opencv <https://clearlinux.org/software/bundle/devpkg-opencv/>`_
+
+   .. tab:: Cloud Orchestration Engineer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - Contains Clear Linux\* OS native software for cloud.
+           - `ethtool <https://clearlinux.org/software/bundle/ethtool/>`_
+
+         * - Utilities for controlling TCP/IP networking and traffic control.
+           - `iproute2 <https://clearlinux.org/software/bundle/iproute2/>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+         * - C++ runtime support.
+           - `libstdcpp <https://clearlinux.org/software/bundle/libstdcpp/>`_
+
+         * - Load and enumerate PKCS#11 modules.
+           - `p11-kit <https://clearlinux.org/software/bundle/p11-kit/>`_
+
+   .. tab:: Game Developer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - All packages required to build the games bundle.
+           - `games-dev <https://clearlinux.org/software/bundle/games-dev/>`_
+
+         * - Godot game engine editor.
+           - `Godot <https://clearlinux.org/software/flathub/godot/>`_
+
+         * - Real-time game engine (Flatpak).
+           - `Unity Hub <https://clearlinux.org/software/flathub/unity-hub/>`_
+   
+         * - Classic point and click adventure game engine and (Flatpak).
+           - `adventure editor <https://clearlinux.org/software/flathub/adventure-editor/>`_
+
+   .. tab:: Kernel Developer
```

Comment:
@bwarden  Could you please review and advise on the bundles I included? My goal is to list at least 3 bundles per profile. Suggest removals too as appropriate.

Reply:
I think the kernel developer section in general is going to have very few suggestions. If you install any `-dkms` bundle, you'll get all the tools you need to do kernel development. I think `kernel-install` is part of the base file system.